vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridTooltipIT.java

```diff
@@ -74,6 +74,58 @@ public void dynamicallyAddGenerator_hoverOverTooltipColumnCell_showTooltip() {
         Assert.assertEquals("Age of the person is 33", getActiveTooltipText());
     }
 
+    @Test
+    public void addGeneratorForAllGrid() {
+        var grid = $(GridElement.class).id("grid-with-tooltips");
+        scrollToElement(grid);
+        clickElementWithJs("set-grid-tooltip-button");
+        for (int col = 0; col < 13; col++) {
+            for (int row = 0; row < 2; row++) {
+                String firstName = row == 0 ? "Jack" : "Jill";
+                showTooltip(grid.getCell(row, col));
+                // check all columns, except column 5 have grid's tooltip
+                if (col != 5) {
+                    Assert.assertEquals("Grid's tooltip! " + firstName,
+                            getActiveTooltipText());
+                } else {
+                    // check that previously set column (5) tooltip was not
+                    // replaced
+                    Assert.assertEquals(
+                            "First name of the person is " + firstName,
+                            getActiveTooltipText());
+                }
+            }
+        }
```

Comment:
Let's simplify this to check the tooltip of two cells, one using the grid's tooltip, the other using the column's tooltip.

I'd also prefer accessing cells by their contents as done in the other tests. Please also adapt the other added tests to do so.